### PR TITLE
[JENKINS-31153] More renames

### PR DESCRIPTION
**Pull request by jglick**
Things forgotten in #296.

@reviewbybees

### CODE REVIEW

File: cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java [hunk 1]
```diff
@@ -225,7 +225,7 @@ public boolean takesImplicitBlockArgument() {
 
         @Override
         public String getDisplayName() {
-            return "Execute sub-workflows in parallel";
+            return "Execute parts of a Pipeline in parallel";
```
Comment (jglick) [inline cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java]: Noticed in #312.

**Comment by recena** [comment]
:bee:

**Comment by kzantow** [comment]
:bee:

File: TUTORIAL.md [hunk 2]
```diff
@@ -390,23 +390,23 @@ Proceed or Abort
...
-* You will see an unnumbered entry under **master** named  **flowname #10**; executors #1 and #2 on the master are idle.
-* You will also see an entry under your slave, in a numbered row (probably #1) called **Building part of flowname #10**.
+* You will see an unnumbered entry under **master** named  **jobname #10**; executors #1 and #2 on the master are idle.
+* You will also see an entry under your slave, in a numbered row (probably #1) called **Building part of jobname #10**.
 
-Why are there two executors consumed by one flow build?
+Why are there two executors consumed by one Pipeline build?
 
-* Every flow build itself runs on the master, using a **flyweight executor** — an uncounted slot that is assumed to not take any significant computational power.
+* Every Pipeline build itself runs on the master, using a **flyweight executor** — an uncounted slot that is assumed to not take any significant computational power.
```
Comment (oleg-nenashev) [inline TUTORIAL.md]: pipeline run?

File: basic-steps/CORE-STEPS.md [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
 
-In many cases these build steps would be valuable to use from workflows, but it would be overkill to define a separate Workflow-only step.
-Therefore selected build steps can be called directly from workflows.
+In many cases these build steps would be valuable to use from Pipelines, but it would be overkill to define a separate Pipeline-only step.
+Therefore selected build steps can be called directly from Pipelines.
 
 # Syntax
 
-As an example, you can write a flow:
+As an example, you can write a pipeline script:
```
Comment (oleg-nenashev) [inline basic-steps/CORE-STEPS.md]: uppercase "p"

File: cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java (hunk 1 above)
Comment (oleg-nenashev) [inline cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java]: Why not just "Execute in parallel"? The current wording looks bad

**Comment by oleg-nenashev** [comment]
Not-mergeable

File: TUTORIAL.md (hunk 2 above)
Comment (jglick) [inline TUTORIAL.md]: I think we are fine with the term “build” in this context; the question of whether we should introduce a new generic term to correspond to `WorkflowRun` was never really settled.

File: cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java (hunk 1 above)
Comment (jglick) [inline cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java]: Yeah that would work too.

File: basic-steps/CORE-STEPS.md (hunk 3 above)
Comment (recena) [inline basic-steps/CORE-STEPS.md]: @oleg-nenashev Well catched!

File: cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java (hunk 1 above)
Comment (recena) [inline cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java]: I prefered `Execute parts of a Pipeline in parallel` but...

**Comment by amuniz** [comment]
:bee:

**Comment by recena** [comment]
:bee:

**Pull request merged**